### DISCUSSION TITLE
sidebar.root remains displayed as is in the menu

### DISCUSSION THREAD

**Discussion by FabianOSMOND** [requarks/wiki#7316] [Error / Bug Report]
Hello all !

The entire Wiki display is displayed correctly except for sidebar.root which remains as is.
Languages ​​work correctly.
I tried to change the language but in the menu no change sidebar.root is not displayed as (root) or (racine) for the French version.
Does anyone have an idea?

Wiki.js 2.5.303

Thank you by advance,

Fabian

**Comment by dawidp** [comment]
I got the same problem with menu not updating. There is whole navigation section in admin menu where you have to configure it

![obraz](https://github.com/user-attachments/assets/86691b4a-9483-4814-b298-f20d4c86a4d1)

**Comment by FabianOSMOND** [comment]
![image](https://github.com/user-attachments/assets/764ae36c-5199-4052-9999-c59cf4562f3c)

From my experience, stuck like this but submenus items are good !

**Reply by dawidp** [reply]
<s>this might be hardcoded </s>

it's should be translated, found [this][1]:

```typescript
this.currentParent.title = `/ ${this.$t('common:sidebar.root')}`
```
[1]: https://github.com/requarks/wiki/blob/d1b4c8c407961f328fb74492ec03b69c912daa9b/client/themes/default/components/nav-sidebar.vue#L225

**Reply by FabianOSMOND** [reply]
Yes but with older version, everything was fine
Sounds like a bug ?
Il looked in POSTGRESQL the json of lang > Translation is present

**Reply by FabianOSMOND** [reply]
I was able to reproduce the behavior.
When I am on a page, if I click on PRINT, and then I return to reading mode it redisplays (root)
As soon as I resume browsing it goes back to sidebar.root

Another element, when I change the startup screen background (login page) in the administrator / security area.
The login screen becomes black and does not display the new image.
The icon at the top left no longer appears (the one configured in GENERAL / Logo)

**Reply by bandlaw12** [reply]
I'm having this bug too:
<img width="314" alt="Screenshot 2024-08-23 at 2 13 54 PM" src="https://github.com/user-attachments/assets/05b85d67-a05f-43ba-b638-b0d7ac1720cf">

Desired output would look something like this:
<img width="314" alt="Screenshot 2024-08-23 at 2 13 54 PM" src="https://github.com/user-attachments/assets/52a2268f-fec5-44b5-81bc-192ad4c7f142">

**Reply by FabianOSMOND** [reply]
@bandlaw12 if you change the screen Connection background, does it bug ?
Logo disaspear and background is black?
When you are on login screen

**Reply by FabianOSMOND** [reply]
Wich version do you use ?

**Reply by FabianOSMOND** [reply]
@markspolakovs : I Connect to POSTGRESQL
Traduction is right !

**Reply by bandlaw12** [reply]
Running 2.5.303 on a regular theme - not sure what you mean by Connection Background?

**Comment by FabianOSMOND** [comment]
We run the same version.

for the backgroung login page, in administration go to SECURITY and change the backgroung image with one you upload.
Disconnect from your account to be redirect on login page.
For me it works one Time and after the backgroung gone black and the logo too….

sound like bugs

**Reply by bandlaw12** [reply]
Yes, clearing the cache and logging out /back in changed it slightly, now mine says / (root) instead of sidebar.root, so that's an improvement I suppose. Definitely a minor bug.

**Comment by hongliangzheng11** [comment]
![1111](https://github.com/user-attachments/assets/f3773b72-6543-43cc-b1cf-d28da112b6bb)
I'm having this bug too.
I'm really wanna how can i fix it.

**Comment by vaxr** [comment]
Hi! Any progress on this bug?

**Comment by prohtex** [comment]
Us too

**Comment by LagClear** [comment]
Same!

**Comment by ThijmenGThN** [comment]
Created a temporary patch.

![image](https://github.com/user-attachments/assets/08d4b133-fc4a-471b-8c2a-31bae3f2ab98)

**Theme > Code Injection > Body HTML-injection**

```js
<script>

const sidebarText = "Return";

(() => {const updateElement = el => {if(el.innerHTML === '/ sidebar.root' || el.innerHTML === '/ (root)'){el.innerHTML = sidebarText; console.log("Element updated to:", el.innerHTML);}}; const checkElement = () => {const targetEl = document.querySelector("#app > div.v-application--wrap > nav > div.v-navigation-drawer__content > div > div > div > div:nth-child(1) > div.v-list.py-2.v-sheet.theme--dark.v-list--dense.primary > div:nth-child(1) > div.v-list-item__title"); if(targetEl){updateElement(targetEl);} requestAnimationFrame(checkElement);}; if(document.readyState === 'loading'){document.addEventListener('DOMContentLoaded', () => requestAnimationFrame(checkElement));} else {requestAnimationFrame(checkElement);}})();
</script>
```

**Reply by DihaHub** [reply]
This does not work for me anymore in the latest version release 2 weeks ago. Maybe because of locale de we use?

**Reply by pat-the-coder** [reply]
Go to Admin → Theme (as an admin), scroll to Injected Code, and paste this into the Body field (not Head — it needs the DOM to exist):

```
<script>
(function () {
  function fixSidebarRoot () {
    document.querySelectorAll('.v-list-item__title').forEach(function (el) {
      if (el.textContent.trim() === '/ sidebar.root') {
        el.textContent = '/ (root)'
      }
    })
  }
  var observer = new MutationObserver(fixSidebarRoot)
  observer.observe(document.body, { childList: true, subtree: true, characterData: true })
  fixSidebarRoot()
})()
</script>
```

Save, then hard-refresh. This watches the DOM and swaps the untranslated key for the correct label whenever the race condition hits — it's cosmetic (patches the symptom, not i18next's load order)``

**Comment by ge9** [comment]
Same bug for me.
I found "sidebar.root" in assets/js/theme0.js and removed it to show `/` instead or `/sidebar.root`.

**Comment by a-moscatelli** [comment]
also see #4888

**Comment by deroguerre** [comment]
same bug
